src/Model/Resolver/Aggregations.php

```diff
@@ -53,4 +82,27 @@ private function processPriceFilter(array $result): array {
             return $item;
         }, $result);
     }
+
+    /**
+     * Process options and replace '1' and '0' labels for options having boolean type.
+     * @param array $result Filters
+     * @return array
+     */
+    protected function enhanceAttributes(array $result): array {
+        foreach ($result as $attr => $attrGroup) {
+            $attribute = $this->attribute->loadByCode('catalog_product', $attrGroup['attribute_code']);
+
+            // Add flag to indicate that attribute is boolean (Yes/No, Enable/Disable, etc.)
+            if ($attribute->getFrontendInput() == 'boolean') {
```

Comment:
This `if` can be done inline:
`$result[$attr]['is_boolean'] = $attribute->getFrontendInput() === 'boolean'`

P.S. please always do strict comparison (`===` instead of `==`)